homeassistant/components/switchbot/binary_sensor.py

```diff
@@ -25,12 +25,10 @@
     ),
     "motion_detected": BinarySensorEntityDescription(
         key="pir_state",
-        translation_key="motion",
```

Comment:
This does not work as you think it does.

Right now, the resulting entity name will be "Motion", as it will inherit it from the device class (as the integration doesn't provide a name or translation key).

So, what is exactly the goal? Making it the device name, or using translations provided by the device class?

Reply:
In that case, yes, it needs to be set to None